.gitignore

```diff
@@ -24,6 +24,7 @@ build/
 bld/
 [Bb]in/
 [Oo]bj/
+localpkg/
```

Comment:
I don't even know where those are coming from. @karajas do you think this is a side-effect of your build tools change?